brats/preprocess.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+import nibabel as nib
+from argparse import ArgumentParser
+from torchvision.transforms import Compose
+
+from brats.transformations import HistogramMatchingTransform
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--dataset_path', type=str, help='Path to the folder '
+                                                         'with patients')
+    parser.add_argument('--template_path', type=str, help='Path to the patient'
+                                                          'which will be used'
+                                                          'as a template for '
+                                                          'histogram matching')
+    parser.add_argument('--output_dir', type=str, help='Directory for '
+                                                       'preprocessed data')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    template = nib.load(args.template_path)
+    template = template.get_fdata()
+    transforms = Compose([HistogramMatchingTransform(template)])
+    for dir_entry in list(os.scandir(args.dataset_path)):
```

Comment:
Is there any reason for calling `list` on top of `os.scandir(args.dataset_path)`?

Reply:
Nope, I was using some other function previously and forgot to remove it.